apko/private/apko_image.bzl

```diff
@@ -6,6 +6,7 @@ load("@bazel_skylib//lib:paths.bzl", "paths")
 _ATTRS = {
     "contents": attr.label(doc = "Label to the contents repository generated by translate_lock. See [apko-cache](./apko-cache.md) documentation.", mandatory = True),
     "config": attr.label(doc = "Label to the `apko.yaml` file.", allow_single_file = True, mandatory = True),
+    "resolved": attr.label(doc = "Label to the `apko.resolved.json` file.", allow_single_file = True, mandatory = True),
```

Comment:
apko_image doesn't need to take the `resolved` file again, as this could be provided by  `contents` with an outputgroupinfo provider. 

I'd like to keep the API surface clean.

Reply:
Done. PTAL.